doc/classes/String.xml

```diff
@@ -742,23 +816,29 @@
 			<param index="0" name="delimiter" type="String" />
 			<param index="1" name="allow_empty" type="bool" default="true" />
 			<description>
-				Splits the string in floats by using a delimiter string and returns an array of the substrings.
-				For example, [code]"1,2.5,3"[/code] will return [code][1,2.5,3][/code] if split by [code]","[/code].
-				If [param allow_empty] is [code]true[/code], and there are two adjacent delimiters in the string, it will add an empty string to the array of substrings at this position.
+				Splits the string into floats by using a [param delimiter] and returns a [PackedFloat32Array].
+				If [param allow_empty] is [code]true[/code], and there are adjacent delimiters in the string, it will add an empty string to the array of substrings at this position.
+				If [param allow_empty] is [code]false[/code], empty or invalid [float] conversions between adjacent delimiters are excluded.
+				[codeblock]
+				var a = "1,2,4.5".split_floats(",")         # a is [1.0, 2.0, 4.5]
+				var c = "1| ||4.5".split_floats("|")        # c is [1.0, 0.0, 0.0, 4.5]
+				var b = "1| ||4.5".split_floats("|", false) # b is [1.0, 4.5]
+				[/codeblock]
 			</description>
 		</method>
 		<method name="strip_edges" qualifiers="const">
 			<return type="String" />
 			<param index="0" name="left" type="bool" default="true" />
 			<param index="1" name="right" type="bool" default="true" />
 			<description>
-				Returns a copy of the string stripped of any non-printable character (including tabulations, spaces and line breaks) at the beginning and the end. The optional arguments are used to toggle stripping on the left and right edges respectively.
+				Strips all non-printable characters from the beginning and the end of the string. These include spaces, tabulations ([code]\t[/code]), and newlines ([code]\n[/code] [code]\r[/code]).
+				If [param left] is [code]false[/code], ignores the string's beginning. Likewise, if [param right] is [code]false[/code], ignores the string's end.
```

Comment:
Random remark since the parameter is literally called "left", calling it "the string's beginning" doesn't acknowledge right-to-left scripts, where the characters removed are still on the left side. I've noticed this in a few places already.

Reply:
It actually was for the opposite reason. Calling out the "string's beginning" was to acknowledge right-to-left.

One could make the argument that the method `left` and `right` are misnamed for that reason, but it's easier to think about using left and right directions.

Reply:
I still don't quite understand how this work, but I trust you if you've thought about it.